module6/lessons/Week5/PreppingForInterviews.md

```diff
@@ -0,0 +1,102 @@
+You've busted your butt and landed an interview! GREAT! Now what?
+
+Each interview deserves its own preparation.  Try to give yourself at least a few days before the interview so that you are fully prepared and confident on the day.
+
+# Prepping for an Interview - A Checklist!
+
+## Review the Job Description
+It might have been a while since you really read through the job description.  Make sure you do that before the interview so that you can:
+* prep for the correct tech stack
+* highlight your relevant skill and experience
+
+## Research the Company
+Before the interview, you will want to have a good understanding of the company as a whole.  You should know:
+* what products they offer
+* recent news about the company
+* the mission, values and culture of the company
+
+This may (probably will) require you to reach out to folks at the company - a good opportunity for coffee chats!
+
+## Practice Your Answers
+We have been doing this generally during Mod6, but you should practice answering questions that you might encounter that are more *specific* to the company/role. The best way to practice an interview is to have someone else ask you questions.  Reach out to your cohort-mates to schedule mock interviews! 
+
+Some things to think about when asking for mock interviews:
+* You can send the questions that you want to be asked - these should be specific to the tech stack or company culture for this role.
+* Treat this as a real interview - practice it as you will complete it (dress appropriately, gracefully exit, etc...)
+
+## Dress Appropriately
+### Dress Yourself
+Make sure you dress appropriately for the interview, taking into account the company's culture and dress code. It's always better to err on the side of dressing too professionally, rather than too casually.
+
+It is always ok to ask the interviewer what the company dress code is!
+
+### Dress Your Space (Remote Interview)
+If you are interviewing remotely - make sure your space is set up appropriately.
+* Clean up any visible messes in your background.
+* Make sure everything in your background reflects what you *want* the interviewer to see.
+* Use a zoom background!  If you're not sure of the company culture, or are unable to put yourself into a good environment for the interview, use a zoom background.
+
+## Arrive Early
+You never know what might come up on the day of the interview, so plan to arrive 10-15 minutes early to account for traffic/parking/check-in.  Even if you are interviewing remotely, you should click on the join link **EARLY**; you never know when zoom might need to update!
+
+## Bring Supplies
+If you are interviewing in person, make sure to bring:
+* Note-taking supplies
+* Fully charged computer
+* A few copies of your resume
+
+If you are interviewing remotely, make sure you have handy:
+* A copy of your resume
+* note-taking supplies
+* Projects that you want to highlight pulled up and ready to highlight!
+
+<section class='call-to-action' markdown='1'>
```

Comment:
I like this activity!

I think makes most sense to move this section to the end. And put the talking about coding first. That way after introducing the activity we can break for independent work.

Reply:
I agree, reordering makes more sense!